plugin/mutate/source/dextool/plugin/mutate/backend/database/type.d

```diff
@@ -60,6 +60,7 @@ struct MutationEntry {
     MutationPoint mp;
     MutantTimeProfile profile;
     Language lang;
+    MutationStatusId st_id;
```

Comment:
change `st_id` to `stId` to follow the name standard.
sql queries are snake_case while D code is camelCase.

Reply:
Should I change all the places where you use snake_case too in the D code or just where I use it?